dts/bindings/memory-controllers/st,stm32-fmc-nor-psram.yaml

```diff
@@ -0,0 +1,194 @@
+# Copyright (c) 2022 Georgij Cernysiov
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  STM32 Flexible Memory Controller (NOR Flash/PSRAM/SRAM controller).
+
+  The FMC generates the appropriate signal timings to drive the
+  following types of memories:
+
+  * Asynchronous SRAM and ROM
+    - 8 bits
+    - 16 bits
+    - 32 bits
+  * PSRAM (Cellular RAM)
+    - Asynchronous mode
+    - Burst mode for synchronous accesses with configurable option to split burst
+      access when crossing boundary page for CRAM 1.5.
+    - Multiplexed or non-multiplexed
+  * NOR Flash memory
+    - Asynchronous mode
+    - Burst mode for synchronous accesses
+    - Multiplexed or non-multiplexed
+
+  A unique Chip Select signal (NE) is used per bank. All the other
+  signals (addresses, data and control) are shared. A wide range of
+  devices is supported through programmable timings.
+
+  Refer to the reference manual for more details.
+
+  The FMC NOR/PSRAM controller is defined below the FMC node and banks are
+  defined as child nodes of the FMC NOR/PSRAM controller node.
+
+  You can enable the controller in device tree as follows:
```

Comment:
Nitpick: 'devicetree' is preferred to 'device tree'.

Reply:
Corrected